README.md

```diff
@@ -1,31 +1,35 @@
 
 ## Getting Started
 
+Install GitLens by clicking `Install` on the banner above, or from the Extensions side bar in VS Code by searching for GitLens.
+
 <p>
   <a title="Watch the GitLens Getting Started video" href="https://www.youtube.com/watch?v=UQPb73Zz9qk"><img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/get-started-video.png" alt="Watch the GitLens Getting Started video" /></a>
 </p>
 
-Install GitLens by clicking `Install` on the banner above, or from the Extensions side bar in VS Code, by searching for GitLens.
+- Use `Switch to Pre-Release Version` on the extension banner to be the first to experience new features.
+
+> Have questions or concerns? Talk to our engineering team directly through our [GitHub Discussions](https://github.com/gitkraken/vscode-gitlens/discussions/categories/feedback) page. Having a positive experience with GitLens? Feel free to [write a review](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens&ssr=false#review-details).
+
+## GitLens Editions: Free and Pro
 
-Use `Switch to Pre-Release Version` on the extension banner to live on the edge and be the first to experience new features.
+**GitLens Free** gives you powerful tools to manage Git and understand how your code has evolved and by whom. With popular features like in-editor blame annotations, hovers, and CodeLens, you can see actionable authorship details at the top of each file. Track the history of any file over time using Revision Navigation to gain deeper insights into code changes.
 
-## Is GitLens Free?
+**GitLens Pro** takes your workflow to the next level by unlocking advanced features and seamless integrations:
 
-All features are free to use on all repos, **except** for features,
+- **Accelerate PR reviews** with Launchpad, a clean and actionable PR hub built directly into VS Code.
+- **Manage commits effortlessly** using the Commit Graph, where you can execute advanced actions like rebase, merge, and more. With powerful search and filtering, quickly locate commits, branches, or files.
+- **Enhance collaboration** by integrating with platforms like GitHub, GitLab, and Bitbucket, reducing context switching. View and manage PRs directly in VS Code through Launchpad.
 
-- marked with a `Pro` require a [trial or paid plan](https://www.gitkraken.com/gitlens/pricing) for use on privately-hosted repos
-- marked with a `Preview` require a GitKraken account, with access level based on your [plan](https://www.gitkraken.com/gitlens/pricing), e.g. Free, Pro, etc
+You can try GitLens Pro for free. When you're ready to upgrade, a GitKraken [paid plan](https://www.gitkraken.com/gitlens/pricing) is required for privately-hosted repos. Some Pro features are available for free on public repos. `Preview` features may require a GitKraken account and could become Pro features in the future. 
```

Comment:
@eatmorespinach did we determine if there was a good URL for making "try GitLens Pro for free" a link to start a sign-in and start a trial that would still be attributable GitLens? Seems like this would be a more worthwhile link than "paid plan" that shows the pricing page.

Reply:
@justinrobots Yep added the GL GK.dev signup link. I was debating whether or not to leave the pricing pg link and that sentence, and went ahead and removed. If there's feedback/thoughts that it should be displayed we can add back in. Go ahead and make any final edits/tweaks here to your discretion as well @justinrobots - I'm good with that. 

I just added this commit here in github - Lmk if you don't see it will let you take it from here.